src/abstractdataframe/reshape.jl

```diff
@@ -171,12 +173,19 @@ unstack(df::AbstractDataFrame)
 
 * `value` : the value column, defaults to `:value`
 
+* `renamecol` : the function that takes a column name from `colkey` and returns
```

Comment:
Maybe something like this? (It would also replace the note below.)
```
a function called on each unique value in `colkey` which must return the name of the column to be created (typically as a string or a `Symbol`). Duplicate names are not allowed.
```

Also, I like `renamecols` better since the function is applied to all columns, and that's the standard suffix we use.

Reply:
OK - fixed. I understand that by

> I would also replace the note below

you wanted me to remove it. Right?